docs_zh-CN/tutorials/customize_runtime.md

```diff
@@ -1 +1,291 @@
 ## 自定义训练配置
+
+### 自定义优化设置
+
+#### 自定义 Pytorch 中的优化器
+
+我们已经支持使用 Pytorch 所有的优化器，并且仅在 config 文件中更改设置即可使用。例如，如果你使用 `ADAM`, 更改如下：
+
+```python
+optimizer = dict(type='Adam', lr=0.0003, weight_decay=0.0001)
+```
+
+为了更改模型的学习率，使用者只需要更改 config 文件中的优化器的 `lr` 参数。使用者可以直接按照 Pytorch 的 `API doc` 设置参数。
+
+#### 自定义自己实现的优化器
+
+#### 1. 定义一个新的优化器
+
+一个自定义的优化器如下：
+假定你增加的优化器为 `MyOptimizer`, 它有参数 a, b, c。你需要建立一个新的文件 `mmtrack/core/optimizer/my_optimizer.py`。
+
+```python
+from torch.optim import Optimizer
+from mmcv.runner.optimizer import OPTIMIZERS
+
+
+@OPTIMIZERS.register_module()
+class MyOptimizer(Optimizer):
+
+    def __init__(self, a, b, c)
+```
+
+#### 2. 将优化器加入注册器中
+
+为了能够找到上述定义的模块，它应首先被引入到主命名空间中。我们提供两种方式来实现：
+
+- 在文件 `mmtrack/core/optimizer/__init__.py` 中引入
```

Comment:
forgot to translate `The newly defined module should be imported in mmtrack/core/optimizer/__init__.py so that the registry will find the new module and add it:`